glue/core/tests/test_data.py

```diff
@@ -38,7 +38,13 @@ def setup_method(self, method):
         self.comp_id = self.data.add_component(comp, 'Test Component')
 
     def test_2d_component_print(self):
-        assert str(self.comp) == 'Component with shape (2, 3)'
+        try:
+            assert str(self.comp) == 'Component with shape (2, 3)'
+        except AssertionError as e:  # Windows
+            try:
+                assert str(self.comp) == 'Component with shape (2L, 3L)'
```

Comment:
minor nitpick: should we change the implementation instead of the test? "2L, 3L" seems potentially awkward